assets/js/blocks/cart-checkout/cart/checkout-button/index.js

```diff
@@ -45,6 +49,20 @@ const CheckoutButton = ( { link } ) => {
 			<PaymentMethodIcons
 				icons={ getIconsFromPaymentMethods( paymentMethods ) }
 			/>
+		</>
+	);
+
+	return (
+		<div className="wc-block-cart__submit">
+			{ positionObserver }
+			<div className="wc-block-cart__submit-container">
+				{ submitContainerContents }
+			</div>
+			{ position === 'below' && (
```

Comment:
> Do we need any fallback for internet explorer? (probably having it scroll into view is fine..)

I don't think so, the sticky behavior is only available on mobile so no need to add support for IE11.

Reply:
Ah - fair point. Though just to confirm, does the sticky behaviour kick in based on window size? If so it can happen on any platform. But I agree, no need for fallback as this is a progressive enhancement.